tidb-cloud/cppo-customer.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Reseller's Customer
+summary: Learn how to become a reseller's customer.
+---
+
+# Reseller's Customer
+
+A reseller's customer subscribes to TiDB Cloud services through a reseller in the AWS Marketplace.
+
+Unlike direct TiDB Cloud customers, a reseller's customers pay invoices via the AWS Marketplace rather than directly to PingCAP. However, all other daily operations in the TiDB Cloud console remain the same for both AWS CPPO and direct TiDB Cloud customers.
```

Comment:
```suggestion
Unlike direct TiDB Cloud customers, a reseller's customers pay invoices via the AWS Marketplace rather than directly to PingCAP. However, all other daily operations in the TiDB Cloud console remain the same for both AWS Marketplace Channel Partner Private Offer (CPPO) and direct TiDB Cloud customers.
```